### PR TITLE
Import Jobs as Projects

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

**Review by labhvam5 — APPROVED** [review]

File: apps/mappings/imports/modules/projects.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    attribute.destination_id
+                ),
+                'is_enabled': True if attribute.active is None else attribute.active
+            }
+
+            # Create a new project if it does not exist in Fyle
+            if attribute.value.lower() not in existing_fyle_attributes_map:
+                payload.append(project)
+            # Disable the existing project in Fyle if auto-sync status is allowed and the destination_attributes is inactive
+            elif is_auto_sync_status_allowed and not attribute.active:
```
Comment (NileshPant1999) [inline apps/mappings/imports/modules/projects.py:56]: where is this function defined @ruuushhh

File: apps/mappings/imports/modules/projects.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        self,
+        paginated_destination_attributes: List[DestinationAttribute],
+        existing_fyle_attributes_map: object,
+        is_auto_sync_status_allowed: bool
+    ):
+        """
+        Construct Fyle payload for Project module
+        :param paginated_destination_attributes: List of paginated destination attributes
+        :param existing_fyle_attributes_map: Existing Fyle attributes map
+        :return: Fyle payload
```
Comment (NileshPant1999) [inline apps/mappings/imports/modules/projects.py:37]: there is a third param too

**Comment by NileshPant1999** [comment]
make sure you tested it locally ??

File: apps/sage300/serializers.py [hunk 3]
```diff
@@ -79,6 +79,8 @@ class Sage300FieldSerializer(serializers.Serializer):
     display_name = serializers.CharField()
 
     def format_sage300_fields(self, workspace_id):
+
+        trigger_import_via_schedule(1, 'ACCOUNT', 'CATEGORY', False)
```
Comment (NileshPant1999) [inline apps/sage300/serializers.py:83]: please remove this this is not needed, its not there in master now

File: tests/test_sage300/test_views.py [hunk 4]
```diff
@@ -27,7 +27,7 @@ def test_sync_dimensions(api_client, test_connection, mocker, create_temp_worksp
     assert response['message'] == 'Sage300 credentials not found / invalid in workspace'
 
 
-def test_sage300_fields(api_client, test_connection):
+def test_sage300_fields(api_client, test_connection, mocker, create_temp_workspace, add_sage300_creds):
```
Comment (NileshPant1999) [inline tests/test_sage300/test_views.py:30]: no need to change here, once you fixed my other comment

**Review by NileshPant1999 — APPROVED** [review]

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**